crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -344,24 +359,25 @@ impl Plugin for VisibilityPlugin {
     fn build(&self, app: &mut bevy_app::App) {
         use VisibilitySystems::*;
 
-        app.configure_sets(
-            PostUpdate,
-            (CalculateBounds, UpdateFrusta, VisibilityPropagate)
-                .before(CheckVisibility)
-                .after(TransformSystem::TransformPropagate),
-        )
-        .configure_sets(PostUpdate, CheckVisibility.ambiguous_with(CheckVisibility))
-        .configure_sets(PostUpdate, VisibilityChangeDetect.after(CheckVisibility))
-        .init_resource::<PreviousVisibleEntities>()
-        .add_systems(
-            PostUpdate,
-            (
-                calculate_bounds.in_set(CalculateBounds),
-                (visibility_propagate_system, reset_view_visibility).in_set(VisibilityPropagate),
-                check_visibility::<With<Mesh3d>>.in_set(CheckVisibility),
-                mark_view_visibility_as_changed_if_necessary.in_set(VisibilityChangeDetect),
-            ),
-        );
+        app.register_type::<VisibilityClass>()
+            .configure_sets(
+                PostUpdate,
+                (CalculateBounds, UpdateFrusta, VisibilityPropagate)
+                    .before(CheckVisibility)
+                    .after(TransformSystem::TransformPropagate),
+            )
+            .configure_sets(PostUpdate, CheckVisibility.ambiguous_with(CheckVisibility))
```

Comment:
Just noticed this while reading, what does it mean for a set to be ambiguous with itself? It was like this before as well.

Reply:
Ambiguities between systems within the set will be ignored. This should probably be removed now!